### PR TITLE
Creating classes and first properties

### PR DESCRIPTION
**Pull request by kmadridr**
Adding doc and classz attributes and methods

### CODE REVIEW

File: src/news-judge-service/app/response.py [hunk 1]
```diff
@@ -1,3 +1,20 @@
+import json
+
 class Response:
+    """
+    This class will hold all the necessary information for the restful response
+    """
     def __init__(self):
-        print("To do");
+        self._status = 0
```
Comment (erickduran) [inline src/news-judge-service/app/response.py:8]: will this be protected? I'd prefer private (use 2 `_`s). Example: `__my_var`.

File: src/news-judge-service/app/Application.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
 from flask import Flask
 from flask_restful import Resource, Api
 from waitress import serve
+import response
```
[the remark this answers is not in the document]
Comment (kmadridr) [reply src/news-judge-service/app/Application.py:4]: Could the standard response class fulfill our needs for this class?

File: src/news-judge-service/app/Application.py [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 app = Flask(__name__)
 api = Api(app)
 
 class Application(Resource):
     def get(self):
-        return {
-            'hello': 'world'
-            }
+        res = response.Response()
+        return res.setStatus(200).setNews(None).build()
```
[the remark this answers is not in the document]
Comment (kmadridr) [reply src/news-judge-service/app/Application.py]: Done

File: src/news-judge-service/app/response.py [hunk 4]
```diff
@@ -1,3 +1,20 @@
...
+        self._news = None
+
+    def setStatus(self, status):
```
[the remark this answers is not in the document]
Comment (kmadridr) [reply src/news-judge-service/app/response.py]: Done

File: src/news-judge-service/app/response.py (hunk 1 above)
Comment (kmadridr) [reply src/news-judge-service/app/response.py]: Done

File: src/news-judge-service/app/Application.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (erickduran) [reply src/news-judge-service/app/Application.py]: No, has a different use case.

File: src/news-judge-service/app/application.py [hunk 5]
```diff
@@ -2,16 +2,19 @@
 from flask_restful import Resource, Api
 from waitress import serve
 import response
+import judge
+import news
```
Comment (erickduran) [inline src/news-judge-service/app/application.py:6]: I'd prefer:
```
from judge import Judge
```
and make the direct references in the code

Comment (kmadridr) [reply src/news-judge-service/app/application.py:6]: Thanks

File: src/news-judge-service/app/ads_validator.py [hunk 6]
```diff
@@ -0,0 +1,6 @@
+from ivalidate import IValidate
```
Comment (erickduran) [inline src/news-judge-service/app/ads_validator.py:1]: `IValidate`? Purpose of the class not clear. If this is just the abstract class for validators, I'd suggest `Validator`.

File: src/news-judge-service/app/application.py [hunk 7]
```diff
@@ -2,16 +2,16 @@
...
 
 app = Flask(__name__)
 api = Api(app)
 
 class Application(Resource):
     def get(self, url):
-        judgy = judge.Judge()
-        post = news.News(url)
+        judgy = Judge()
+        post = News(url)
```
Comment (erickduran) [inline src/news-judge-service/app/application.py:14]: Thanks

File: src/news-judge-service/app/judge.py [hunk 8]
```diff
@@ -1,4 +1,7 @@
-import response
+from response import Response
+from url_validator import UrlValidator
+from ads_validator import AdsValidator
+from ssl_validator import SslValidator
```
Comment (erickduran) [inline src/news-judge-service/app/judge.py:4]: Names of classes with acronyms are named with caps. E. g. `URLValidator`, `SSLValidator`.

Comment (erickduran) [reply src/news-judge-service/app/judge.py:4]: Also, maybe create submodules for validators? E. g. `from validators.url_validator import URLValidator`. Having directories with submodules may help for a more organized code structure.

**Pull request merged**